sections/image-banner.liquid

```diff
@@ -294,7 +304,27 @@
     },
     {
       "type": "header",
-      "content": "t:sections.image-banner.settings.header.content"
+      "content": "t:sections.image-banner.settings.animations.content"
```

Comment:
Mainly wondering if we should include those settings globally instead, to apply at once on the sections we would include. We would revisit the content if this is the case. Let's regroup to align! 🙏